### DISCUSSION TITLE
Error bytes did not verify (flashing MX29LV320, HiROM) [RESOLVED]

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#955] [Need help]
Hi,
I get error when flashing MX29LV320
Verifying Error: 1429472 bytes did not verify

When flashing the same flash for the second time, then I get "Error: 1429147  bytes did not verify" , 
For third time : 1428726
Is it normal?

CART READER V3
software version 7.3

![IMG_6807](https://github.com/sanni/cartreader/assets/38383423/7b31e003-2230-4bf1-93a4-65841cd0e9a6)

![IMG_6808](https://github.com/sanni/cartreader/assets/38383423/11e30a7c-c465-46d7-9f13-9f1ebd9da0f9)

**Comment by PsyK0p4T** [comment]
What behavior do you have with the current v13.2 software version ?

**Reply by StanMOI** [reply]
Update done, but now I get an other error :
https://github.com/sanni/cartreader/discussions/956

**Comment by StanMOI** [comment]
With Version: 13.1
I still get error code when  verifying, now it's : 202676 (and so on...202524)
![IMG_6811](https://github.com/sanni/cartreader/assets/38383423/b93fa05d-222e-4da5-b130-c597eff3506a)

**Comment by StanMOI** [comment]
I done some more tests.
No error with LoROM repro , only with HiROM.
Tried to replace SRAM 64kb by SRAM 256kb = still get the same error code.
Game working on my SNES console.
Is it a bug ?

**Reply by Chomemel** [reply]
Did you try the lastest version of the code? I had the exact same issue and V13.4 solved it (now it's even 13.5). Although, even if I had the verifying error, the games worked on RH before I solved the problem.

**Reply by StanMOI** [reply]
YESSSSS !!!!!
That was it ! 
It was corrected in new update:

- SNES: Fix flashing hirom repros with SRAM

After update to 13.5 version no more error. All OK !
Thank you very much.
![IMG_6915](https://github.com/user-attachments/assets/f6c60824-7afe-4199-b6a8-bedea886b78e)

![IMG_6989](https://github.com/user-attachments/assets/a062f037-0801-44bd-b98c-e1791fe701e0)
